examples/seismic/skew_self_adjoint/wavesolver.py

```diff
@@ -0,0 +1,310 @@
+from devito import Function, TimeFunction
+from examples.seismic import PointSource, Receiver
+from examples.seismic.skew_self_adjoint.utils import *
+from examples.seismic.skew_self_adjoint.operators import *
+
+
+class SSA_ISO_AcousticWaveSolver(object):
```

Comment:
no underscores in class names

Reply:
going to be ugly with all the caps running together, but so be it

Reply:
done